test/e2e/serial/config/config.go

```diff
@@ -67,3 +79,77 @@ func GetRteCiImage() string {
 	}
 	return nropTestCIImage
 }
+
+func contains(items []string, st string) bool {
+	for _, item := range items {
+		if item == st {
+			return true
+		}
+	}
+	return false
+}
+
+var policyScopeToTmPolicy = map[string]nrtv1alpha2.TopologyManagerPolicy{
+	"restricted:pod":             nrtv1alpha2.RestrictedPodLevel,
+	"restricted:container":       nrtv1alpha2.RestrictedContainerLevel,
+	"best-effort:pod":            nrtv1alpha2.BestEffortPodLevel,
+	"best-effort:container":      nrtv1alpha2.BestEffortContainerLevel,
+	"none:":                      nrtv1alpha2.None,
+	"single-numa-node:pod":       nrtv1alpha2.SingleNUMANodePodLevel,
+	"single-numa-node:container": nrtv1alpha2.SingleNUMANodeContainerLevel,
+}
+
+func toTopolyPolicy(topologyManagerPolicy, topologyManagerScope string) (nrtv1alpha2.TopologyManagerPolicy, error) {
+	chain := topologyManagerPolicy + ":" + topologyManagerScope
+	tmPolicy, ok := policyScopeToTmPolicy[chain]
+	if !ok {
+		return "", fmt.Errorf("Unable to convert %q to TmPolicy", chain)
+	}
+	return tmPolicy, nil
+}
+
+func getTopologyConsistencyErrors(kconfigs map[string]*kubeletconfigv1beta1.KubeletConfiguration, nrts []nrtv1alpha2.NodeResourceTopology) map[string]error {
```

Comment:
what if `kconfigs` is a subset of `nrts`?

Reply:
I have added an additional check if there are some NRTs with no kubeletconfiguration.